docs/source/quick_start.md

```diff
@@ -1,13 +1,33 @@
 # Quick Start
 
-While the **pyCoilGen** application can be used as a command-line application, due to the [large number of parameters](./configuration.md) to be configured, it is best used as part of a Python script. 
-
-This allows users to use the application, or previously saved data, to manipulate and/or view coil design parameters, field errors, etc.
+This chapter describes how to get started with using `pyCoilGen`.
 
 ## Calling the Application
 
-The following code snippet shows an example of a basic Python script that creates a dictionary of parameters that is then passed to the `pyCoilGen` function to determine the solution.
+`pyCoilGen` has a command line interface and a Python API.
+
+### Command Line Interface
+
+To call `pyCoilGen` from the command line: 
+
+```
+
+```
+
+For example:
+
+
+```
+
+```
+
+You can access the help with `-h`. All parameters are also described in detail in the [Configuration](./configuration.md) chapter. 
 
+### Python API
```

Comment:
Would a user only need the `pyCoilGen` function?

Reply:
Yes. The application has only this one interface.